backend/controllers/splitController.js

```diff
@@ -0,0 +1,84 @@
+let requestData = {
```

Comment:
optional
I would consider removing this sample message as it is not being used in the code and can be misleading when going through the code.